contractsSol5/KyberMatchingEngine.sol

```diff
@@ -537,8 +552,9 @@ contract KyberMatchingEngine is KyberHintHandler, IKyberMatchingEngine, Withdraw
                 tData.numFeePayingReserves++;
             }
 
-            //take into account possible additional networkFee
+            // this line we can not reach full coverage due to networkFeeWei * 2 + platformFeeWei < BPS and feePayingReservesBps < 2 *BPS
```

Comment:
You can write a maliciousMatchingEngine with a `initAndValidateTradeData` function that doesn't have the BPS checks. Then you'll reach this line(s)

Reply:
change as request